### DISCUSSION TITLE
Maybe the peer is running old software.

### DISCUSSION THREAD

**Discussion by erpehlivan** [Chia-Network/chia-blockchain#4926] [Support]
Hi everyone ,
 its third day i started chia network. but still (not sync ) and i cant fix it.

2021-05-14T10:22:11.139 full_node full_node_server        : WARNING  Invalid handshake with peer. Maybe the peer is running old software.
2021-05-14T10:22:12.161 wallet wallet_server              : WARNING  Trying to ban localhost for 10, but will not ban
2021-05-14T10:22:12.161 wallet wallet_server              : ERROR    Exception list index out of range, exception Stack: Traceback (most recent call last):
  File "chia\server\server.py", line 394, in start_client
  File "chia\server\ws_connection.py", line 113, in perform_handshake
  File "chia\cmds\init_funcs.py", line 300, in chia_full_version_str
  File "chia\cmds\init_funcs.py", line 252, in chia_version_number
IndexError: list index out of range

is there anyone can give advice to solve it ? please

**Comment by Muho747** [comment]
Yeah me to. But I cant do this. I tried everything. Do you fix this? Çözebilirsen bana da anlatabilir misin sana ulaşabileceğim bir link bırak ordan konuşalım. Sorun sadece ikimizde olamaz herhalde..

**Reply by erpehlivan** [reply]
merhaba
 evet çözdüm. users bölümünde ki kullanıcı adı Türkçe karakter vardı .o yüzden olmamış. onu değiştirdim ve sorun düzeldi. Umarım senin ki de basit bir şekilde çözülür

**Reply by Muho747** [reply]
En iyisi havuza bağlanmak hocam değiştirsem de olmadı o kadar saçma ki. Yanımda laptop var o senkronize ediyor ama masaüstü yapmıyor bende solo kazmayı düşünmüyorum plot oluşturup havuza geçtim.

**Reply by Muho747** [reply]
Valla oldu hocam yani saçma geldi ama evet dediğiniz gibi yaptım ve oldu teşekkür ederim:)

**Comment by BrandtH22** [comment]
@erpehlivan , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
